tests/interfaces/records/handlers/records-write.spec.ts

```diff
@@ -1987,23 +1986,25 @@ describe('RecordsWriteHandler.handle()', () => {
     });
   });
 
-  it('should throw if `storageController.put()` throws unknown error', async () => {
-    const { requester, message, dataStream } = await TestDataGenerator.generateRecordsWrite();
-    const tenant = requester.did;
+  // Question for reviewers (mainly thehenrytai): Why are were doing this?
```

Comment:
I think it is a test case for this line of code: https://github.com/TBD54566975/dwn-sdk-js/blob/main/src/interfaces/records/handlers/records-write.ts#L92

The intend for that line of code is so that we don't claim that it is a client error when don't know what that error is and it can very well be a server side error. A case can be made that we should return 500 instead.